packages/typescript-config/tsconfig.json

```diff
@@ -10,11 +10,11 @@
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
     "module": "ESNext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
What is the reasoning behind modeResolution change?

Reply:
And for the removing `incremental: true` 🙂

Reply:
TL; DR; it doesn't work otherwise